pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -1,18 +1,18 @@
 version 1.0
 
-import "../../../tasks/skylab/FastqProcessing.wdl" as FastqProcessing
-import "../../../tasks/skylab/MergeSortBam.wdl" as Merge
-import "../../../tasks/skylab/CreateCountMatrix.wdl" as Count
-import "../../../tasks/skylab/StarAlign.wdl" as StarAlignBam
-import "../../../tasks/skylab/TagGeneExon.wdl" as TagGeneExon
-import "../../../tasks/skylab/SequenceDataWithMoleculeTagMetrics.wdl" as Metrics
-import "../../../tasks/skylab/TagSortBam.wdl" as TagSortBam
-import "../../../tasks/skylab/RunEmptyDrops.wdl" as RunEmptyDrops
-import "../../../tasks/skylab/LoomUtils.wdl" as LoomUtils
-import "../../../tasks/skylab/Picard.wdl" as Picard
-import "../../../tasks/skylab/UmiCorrection.wdl" as UmiCorrection
-import "../../../tasks/skylab/ModifyGtf.wdl" as ModifyGtf
-import "../../../tasks/skylab/OptimusInputChecks.wdl" as OptimusInputChecks
+import "https://raw.githubusercontent.com/broadinstitute/warp/np-hca-adapter-test/tasks/skylab/FastqProcessing.wdl" as FastqProcessing
```

Comment:
Do we actually want to merge this in to Optimus? I would imagine no

Reply:
No we do not! I will update these since they dont need to be http imports anymore